db/migrate/20220619221938_create_user_access_outbox.rb

```diff
@@ -1,6 +1,7 @@
 class CreateUserAccessOutbox < ActiveRecord::Migration[7.0]
   def change
-    create_table :user_access_outboxes, id: :uuid do |t|
```

Comment:
Somehow this was not working for me. So I created a new field `identifier` to be an `uuid`